tests/Microsoft.Identity.Web.Test/TokenAcquisitionAddInTests.cs

```diff
@@ -27,33 +32,61 @@ public void InvokeOnBuildConfidentialClientApplication_InvokesEvent()
             // Assert
             Assert.True(eventInvoked);
         }
-
+#endif
         [Fact]
-        public void InvokeOnBeforeTokenAcquisitionForApp_InvokesEvent()
+        public async Task InvokeOnBeforeTokenAcquisitionForApp_InvokesEvent()
         {
             // Arrange
-            var options = new TokenAcquisitionAddInOptions();
+            var options = new TokenAcquisitionExtensionOptions();
             var acquireTokenOptions = new AcquireTokenOptions();
-            AcquireTokenForClientParameterBuilder builderMock = null!;
+            acquireTokenOptions.ForceRefresh = true;
 
-            bool eventInvoked = false;
-            options.OnBeforeTokenAcquisitionForApp += (builder, options) =>
+            //Configure mocks
+            using MockHttpClientFactory mockHttpClient = new MockHttpClientFactory();
```

Comment:
nit: new can be simplified: `using MockHttpClientFactory mockHttpClient = new();`